homework/index.js

```diff
@@ -29,10 +29,68 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function changeDateFormat(date) {
+    const timeFormat = new Date(date);
+    return timeFormat.toLocaleString();
+  }
+
+  function sortRepos(a, b) {
+    const first = a.name.toLowerCase();
+    const second = b.name.toLowerCase();
+    const key = first < second ? -1 : first > second ? 1 : 0;
+    return key;
   }
 
+  function renderRepoDetails(repo, ul) {
+    const grouping = createAndAppend('li', ul, { class: 'listItem' });
+    const name = createAndAppend('li', grouping, {
```

Comment:
`grouping` is a `<li>`; this can't be the parent of another `<li>`. This would be invalid HTML. You need an intermediate `<ul>` as shown here:

![image](https://user-images.githubusercontent.com/2788771/65444240-ecd93d80-de2f-11e9-84f8-0ada208eee52.png)